go.mod

```diff
@@ -7,10 +7,12 @@ require (
 	github.com/auth0/go-auth0 v0.15.1
 	github.com/briandowns/spinner v1.23.0
 	github.com/charmbracelet/glamour v0.6.0
+	github.com/fsnotify/fsnotify v1.6.0
 	github.com/getsentry/sentry-go v0.19.0
 	github.com/golang/mock v1.6.0
 	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.3.0
+	github.com/guiguan/caster v0.0.0-20191104051807-3736c4464f38
```

Comment:
Should we rely on a dependency that has not been updated since Nov 2019? https://github.com/guiguan/caster